docs/testbed/README.testbed.ocs_data_plan_deployment.md

```diff
@@ -0,0 +1,262 @@
+
+1. Install Ubuntu AMD64 on your host or VM
+    - The host PC needs to have at least 20GB of memory free
+    - If the testbed host is a VM, then it must support nested virtualization
+        - [Instructions for Hyper-V based VMs](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/nested-virtualization#configure-nested-virtualization)
+2. Prepare your environment based on Ubuntu
+    The Ubuntu version now is recommanded with **Ubuntu 20.04**
+    ```
+    sudo apt install python3 python3-pip openssh-server
+    ```
+    If checked the default python version with command `python --version`, the python version on server now is still 2.x. After install python3, replace python command with python3 using symbolic link:
+    ```
+    sudo ln -sf /usr/bin/python3 /usr/bin/python
+    ```
+
+    For more Ubuntu version, running this testbed needs to be explored for compatibility.
+3. Download the latest version of sonic-mgmt on github
+   Create a folder to store the cloned files of sonic-mgmt test scripts.   
+    ```
+    cd ~
+    mkdir SONiC
+    chmod 777 SONiC
+    git clone https://github.com/sonic-net/sonic-mgmt
+    ```
+    Check download script in folder:
+    ```
+    /SONiC# ls
+    sonic-mgmt
+    /SONiC/sonic-mgmt# ls
+    ansible  azure-pipelines.yml  docs  LICENSE  pylintrc  pyproject.toml  README.md  sdn_tests  SECURITY.md  setup-container.sh  sonic_dictionary.txt  spytest  test_reporting  tests
+    ```
+4. [Install Docker CE](https://docs.docker.com/install/linux/docker-ce/ubuntu/). Be sure to follow the [post-install instructions](https://docs.docker.com/install/linux/linux-postinstall/) so that you don't need sudo privileges to run docker commands.
+
+## Download sonic-mgmt docker container
+1. The simplest way way to run the script on testbed is to deploy the official Docker and run cases within the Docker, so there is no need to deploy the required runtime libraries separately.
+   - Build the test enviroment on sonic-vs container. 
+    ```
+    docker pull sonicdev-microsoft.azurecr.io:443/docker-sonic-mgmt:latest
+    ```
+2. Check if the Docker image has been installed on host.
+    ```
+    docker images
+    REPOSITORY                                            TAG       IMAGE ID       CREATED        SIZE
+    sonicdev-microsoft.azurecr.io:443/docker-sonic-mgmt   latest    0550f2d12fde   2 months ago   3.43GB
+    ```
+## Setup sonic-mgmt docker
+All testbed configurations and test procedures execute within a sonic-mgmt Docker container, providing a standardized testing environment with pre-configured packages and tools to ensure consistency of test results on sonic-mgmt framework.
+1. Launch the Docker for running the sonic-mgmt scripts by the following command.
+    ```
+    docker run -d -it -u root -h sonic -v /SONiC:/data:rslave -v "/var/run/docker.sock:/var/run/docker.sock:rslave" --name sonic sonicdev-microsoft.azurecr.io:443/docker-sonic-mgmt
```

Comment:
Recommend to use the `setup-container.sh` tool for setting up sonic-mgmt container.